ops/op2/config.rs

```diff
@@ -0,0 +1,194 @@
+use deno_proc_macro_rules::rules;
+use proc_macro2::TokenStream;
+use proc_macro2::TokenTree;
+use quote::ToTokens;
+
+use crate::op2::Op2Error;
+
+#[derive(Debug, Default, Eq, PartialEq)]
+pub(crate) struct MacroConfig {
+  /// Use `(core)` for ops that live in `deno_core`.
+  pub core: bool,
+  /// Generate a fastcall method (must be fastcall compatible).
+  pub fast: bool,
+  /// Do not generate a fastcall method (must be fastcall compatible).
```

Comment:
```
(must be fastcall compatible).
```

seems wrong here

Reply:
That's actually correct -- `nofast` is only for potentially fast methods.